packages/pkg/src/rules/sort.ts

```diff
@@ -118,9 +266,18 @@ const uniqueArray = <T>(arr: readonly T[]) => {
 }
 
 export const sort = (props: ObjectProperty[], options: FormatOptions) => {
-  let { packageSortOrder } = options
+  let { packageSortOrder, packageSortOrderPreset } = options
+
+  const defaultSortOrder = {
+    npm: NPM_SORTS,
+    'npm-plus': NPM_PLUS_SORTS,
+    unts: UNTS_SORT,
```

Comment:
> I think we don't need a `unts` preset, because there are other properties not listed in npm.

Reply:
So you want to move the full unts list into your own config? Or follow one of the new presets?

Reply:
Nope, it should still be the default, `...([PRESETS[preset] ?? []), ...primary]`